### DISCUSSION TITLE
Rotki is now available in Umbrel!

### DISCUSSION THREAD

**Discussion by Xosten** [rotki/rotki#5125] [Archive]
Hey Rotki fam!

For all Umbrel users out there, Rotki is now available on their App Store! 

Umbrel is an OS for running a personal server in your home. Self-host open source apps like Nextcloud, Bitcoin node, and more. You get the convenience of cloud, without giving up your data. You just need a small computer, VM or a Raspberry Pi! Learn more over [here](https://umbrel.com/)

If you wish to double-check the app from their App Store, you can look over this PR https://github.com/getumbrel/umbrel-apps/pull/255.

**Comment by LefterisJP** [comment]
Hey @Xosten thank you so much!

Really appreciate you doing this. It's so cool to see rotki in umbrel. :bow: 

Is there any way to get a glimpse of the app store before installing umbrel somewhere?

**Reply by Xosten** [reply]
Hey @LefterisJP!

With great pleasure!

You can have a look at the current app store here https://github.com/getumbrel/umbrel-apps. Their apps repository keeps growing!
